WordPress/src/main/java/org/wordpress/android/util/publicdata/WordPressPublicData.kt

```diff
@@ -21,4 +21,18 @@ class WordPressPublicData @Inject constructor() {
         Wasabi.type -> Wasabi.value
         else -> throw IllegalArgumentException("Failed to get Jetpack package ID: build flavor not found.")
     }
+
+    fun currentPackageVersion() = packageManagerWrapper.getPackageInfo(currentPackageId())?.versionName
```

Comment:
Nitpick: would it make sense to add the explicit return type for this public function?

Reply:
Good point 👍 
Added with 398ff12fea92a290a91fc01995b04d101a892ef1